### PR TITLE
Nuke: Expose write knobs - OP-7592

### PR DESCRIPTION
**Pull request by tokejepsen**
## Changelog Description
This PR adds `exposed_knobs` to the creator plugins settings at `ayon+settings://nuke/create/CreateWriteRender/exposed_knobs`.

When exposed knobs will be linked from the write node to the outside publish group, for users to adjust.

## Testing notes:
1. Add a knob to `ayon+settings://nuke/create/CreateWriteRender/exposed_knobs`.
2. Create write instance in Nuke.
3. Check knob is exposed.
4. Add a different knob to `ayon+settings://nuke/create/CreateWriteRender/exposed_knobs`.
5. Publish and check validate kicks in to add the linked knob.

### CODE REVIEW

**Comment by tokejepsen** [comment]
Docs at https://github.com/ynput/ayon-documentation/pull/119

**Review by moonyuet — APPROVED** [review]
Followed the testing notes and it works 
image
Maybe the ` Exposed Knobs` can be more precise like `Exposed Knobs for Write Node` or `Exposed Knobs for Inner Write Node` because I thought it also allows to expose the parameter of the reformat node inside the `WriteNodeInstance`. 
Otherwise, it looks good.

**Comment by tokejepsen** [comment]
> Maybe the  Exposed Knobs can be more precise like Exposed Knobs for Write Node or Exposed Knobs for Inner Write Node because I thought it also allows to expose the parameter of the reformat node inside the WriteNodeInstance.
Otherwise, it looks good.

Good point! Will update to `Write Node Exposed Knobs`.

File: server_addon/nuke/server/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.1.8"
+__version__ = "0.2.8"
```
Comment (jakubjezek001) [inline server_addon/nuke/server/version.py:1]:
```suggestion
__version__ = "0.1.9"
```

**Comment by jakubjezek001** [comment]
> Good point! Will update to `Write Node Exposed Knobs`.

Also I would add devider line with label, something like this
image

**Review by jakubjezek001 — COMMENTED** [review]
Tested the functionality and works well!

I am not only sure how to deal with typos in the input fields like this:
image

**Comment by tokejepsen** [comment]
> Maybe the  Exposed Knobs can be more precise like Exposed Knobs for Write Node or Exposed Knobs for Inner Write Node because I thought it also allows to expose the parameter of the reformat node inside the WriteNodeInstance.
Otherwise, it looks good.

@moonyuet I've change the label to `Write Node Exposed Knobs`.

> Also I would add devider line with label, something like this

@jakubjezek001 done.
Capture

> I am not only sure how to deal with typos in the input fields like this:

@jakubjezek001 I'm raising an error for the user to be warned and directed to the settings to resolve. I would suggest we get https://github.com/ynput/OpenPype/pull/6179 merged, before deploying to client, so there will be an error message pop instead of just script editor output.

**Review by moonyuet — APPROVED** [review]
Looks good for me.
image

**Review by jakubjezek001 — APPROVED** [review]
works well in Ayon!
image

Also if typo in knob name also pops up this warning:
image

**Pull request merged**